google/cloud/BUILD.bazel

```diff
@@ -27,12 +27,32 @@ capture_build_info(
     ],
 )
 
+config_setting(
+    name = "have_open_telemetry",
+    flag_values = {
+        "//:experimental-open_telemetry": "true",
+        "@io_opentelemetry_cpp//api:with_abseil": "true",
+    },
+)
+
+config_setting(
```

Comment:
Maybe:

```
config_setting(
    name = "incompatible_open_telemetry",
    flag_values = {"@io_opentelemetry_cpp//api:with_abseil": "false"},
)

config_setting(
    name = "enable_open_telemetry",
    flag_values = {
         "//:experimental-open_telemetry": "true",
         "@io_opentelemetry_cpp//api:with_abseil": "true",
    },
)
```

Reply:
Hmm... we wouldn't be using the `incompatible_open_telemetry` setting anywhere. I do like "enable" more than "have".

I think the weird thing initially is that I used `have_open_telemetry` and `not_have_open_telemetry`, but they weren't strictly opposites.

So I went with `enable_open_telemetry_valid` and `disable_open_telemetry`. Thoughts?

Reply:
Hmm... still a bit weird. :shrug: your call.